### ISSUE TITLE
Add aria-label to delete button in Grid dynamic height example

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#5764]
The delete button in the [dynamic height example](https://vaadin.com/docs/latest/components/grid#dynamic-height) for Grid lacks an `aria-label`.
